### ISSUE TITLE
"Choose a file" fixes

### ISSUE THREAD

**Issue by adriannelee** [cds-snc/notification-planning#1229]
## Description

As a Dragon Speaking Naturally user, I need to be able to do select the "Choose a file" verbally so that I don't waste my time doing a manual select using window panes. 

As a screen reader user, I need to have better expectations when I select the "Choose a file" button so that I'm not surprised when I move to the next screen

WHY are we building?
- The "Choose a file" button is wastes some time for dictation software users ["Choose a file" is not selectable by voice](https://airtable.com/appWwAmHwDLtpIyko/tbl38n7ofWYBuezFc/viwBYyaQxOttZ0IAw/recZF5ITb6SxEkDHR?blocks=hide) 
-  Using the button is a little surprising for screen reader users because the button's ARIA label is "No file chosen." Screen reader users are more accustomed to hearing an 'update' message to say "1 file chosen" after they make a selection (without moving to another screen/step). ["Browse a file" is clearer. ](https://airtable.com/appWwAmHwDLtpIyko/tbl38n7ofWYBuezFc/viwBYyaQxOttZ0IAw/recZWf2qXyuZs9501?blocks=hide)

WHAT are we building?
- Fixes to the "Upload a file" button

VALUE created by our solution
- It's easier and more predictable for public servants using assistive tech to send messages using spreadsheets.

## Documentation and Artifacts
[Clip of dictation software user](https://docs.google.com/presentation/d/1Rm8MSOnjLB08WW1spb5pitx56WO9eyEcNtl5Cat_akU/edit#slide=id.g245d729d97d_0_9)

Good docs, figma mockups, ADRs, screenshots etc.

## Acceptance Criteria
Given some context, when (X) action occurs, then (Y) outcome is achieved 

- [ ] When using a dictation software, "Upload a file" is selectable by voice
- [x] When using a screen reader, it's more clear that they've selected their spreadsheet and are moving on to the next step/screen
- [ ] Update the status of related findings, insights, and hypotheses on the Research Airtable
- [ ] Once change/fix/feature is implemented, link relevant Airtable records to design artifacts (Figma)

## Related [Research Airtable](https://airtable.com/appWwAmHwDLtpIyko/tbl38n7ofWYBuezFc/viwQUX3tUxbMxYS7C?blocks=hide) records
- See above links

## QA Steps
- [ ] Tested in a realistic production scenario

**Comment by amazingphilippe** [comment]
[Fable testers confirmed the same issues identified here. ](https://cds-snc.dovetailapp.com/insights/3ljy9QYKrsKVwnc7beMCBS)

**Comment by YedidaZalik** [comment]
Adding to Ally stmt

**Comment by andrewleith** [comment]
It might be nice to test our request branding page with fable users, since we implemented the file upload feature a bit there and it should be a little better now!

**Comment by andrewleith** [comment]
Lets wait for our contract to renew with fable and start there.

**Comment by amazingphilippe** [comment]
We've got PR 1807 above to test with too now.

**Comment by amazingphilippe** [comment]
We got some results back on the file upload buttons prototypes

Both options performed good and/or poorly for all 5 testers. Which just shows how a file upload, even as a native html element can be tricky to interact with. 

Everyone was able to complete the task.

Options: 
- We can book some follow-ups to figure out what else we can do at this point, because we don’t have a clear winner, although everyone was able to complete the tasks.
- We can pick one based on other criteria (like being able to choose and localize the button label)

**Comment by yaelberger-commits** [comment]
Andrew found: 
i just found this old issue from GDS about file inputs and dragon (https://github.com/alphagov/govuk-frontend/issues/2425) 
They also created some prototypes here: https://codepen.io/matteason/pen/poWqjdR

**Comment by yaelberger-commits** [comment]
Bringing back to Sprint Backlog until @amazingphilippe is back

**Comment by amazingphilippe** [comment]
PR with the most recent Dragon fix ready for review: https://github.com/cds-snc/notification-admin/pull/1807

Also cleaned up prototypes and debug out of it. Its ready for release in my eyes, given we do a fresh code review.

**Comment by andrewleith** [comment]
- ready for re-review: @whabanks, @andrewleith

**Comment by andrewleith** [comment]
- Will re-run ui tests now that branding tests have been merged
